puppet/modules/utility/manifests/init.pp

```diff
@@ -7,9 +7,7 @@
 
   stdlib::ensure_packages([$vim])
 
-  unless $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8' {
-    stdlib::ensure_packages(['htop', 'iftop', 'screen'])
-  }
+  stdlib::ensure_packages(['htop', 'iftop', 'screen'])
```

Comment:
`screen` is back in EL9?

Reply:
No. Which makes me wonder if the `utility` class is properly applied, as the EL9 boxen do not drop any errors but that should have been applied there already?

Reply:
There's a very good chance I somehow missed this. It's supposed to be applied via the Foreman ENC because we don't have the admins in Hiera (for privacy reasons). https://github.com/theforeman/foreman-infra/pull/1789 was intended to solve that but as you can see, that's been open for a while.

Reply:
We could add `utility` to the `base` config group, but that'd mean we need EPEL on all (EL) systems, which we today don't do.